src/Symfony/Bridge/Twig/Resources/views/Form/form_div_layout.html.twig

```diff
@@ -1,19 +1,19 @@
 {# Widgets #}
 
-{% block form_widget -%}
+{%- block form_widget -%}
     {% if compound %}
         {{- block('form_widget_compound') -}}
     {% else %}
         {{- block('form_widget_simple') -}}
     {% endif %}
-{%- endblock form_widget %}
+{%- endblock form_widget -%}
```

Comment:
for the end and the beginning of blocks, it does not matter given that we are only rendering blocks, not the template itself